package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sap_oss/wdio-qmate-service",
-  "version": "2.9.8",
+  "version": "2.9.9",
   "lockfileVersion": 3,
```

Comment:
@cvakiitho gh-action complains about the lockversion. But actually it didn't get changed?